File: AtlasWeb/Core/Modules/Tibbi_Surec/Hasta_Islemi_Goruntuleme/EpisodeActionDisplayFormViewModel.cs
using System;
using System.Collections.Generic;
using TTObjectClasses;

namespace Core.Models
{
    public class EpisodeActionDisplayFormViewModel
    {
        public string ObjectID;
        public string ObjectDefName;
        public string FormDefID;
        public string InputParam;

    }
    public class ActiveInfoDVO
    {
        public Guid EpisodeActionObjectID;
        public Guid EpisodeObjectID;
        public Guid PatientObjectID;
    }
}

File: AtlasWeb/Core/Modules/Tibbi_Surec/Hasta_Islemi_Goruntuleme/EpisodeActionDisplayFormViewModel.ts
import { BaseModel } from 'Fw/Models/BaseModel';
import { Guid } from 'NebulaClient/Mscorlib/Guid';

export class EpisodeActionDisplayFormViewModel extends BaseModel {

    public ObjectID: string;
    public ObjectDefName: string;
    public FormDefID: string;
    public InputParam: any;

}

export class ActiveInfoDVO {
    public EpisodeActionObjectID: Guid;
    public EpisodeObjectID: Guid;
    public PatientObjectID: Guid;
}